src/test/cypress/cypress/support/feedCommands.js

```diff
@@ -44,4 +52,40 @@ export class FeedCommands extends OpfabCommands {
         cy.get('#opfab-card-edit').click();
         cy.get("of-usercard").should('exist');
     }
+
+    sortByUnread() {
```

Comment:
**sortByRead = function() {**

Same for all , other wise the mechanism of adding code for logging does not take into account the method